config/settings.yml

```diff
@@ -1019,6 +1019,14 @@ lighthouse:
     access_token:
       aud_claim_url: https://deptva-eval.okta.com/oauth2/ausi3ui83fLa68IJv2p7/v1/token
     use_mocks: false
+  benefits_education:
+    timeout: 30
```

Comment:
Curious how this timeout affects this integration with Lighthouse?

Reply:
That setting should've been removed when I was cleaning the code up for the PR.  It's removed now.